integration/v7/isolated/continue_deployment_test.go

```diff
@@ -87,7 +87,7 @@ var _ = Describe("Continue Deployment", func() {
 			When("There is a canary deployment", func() {
 				It("succeeds", func() {
 					helpers.WithHelloWorldApp(func(appDir string) {
-						helpers.CF("push", appName, "-p", appDir, "--strategy=canary", "--no-wait").Wait()
+						helpers.CF("push", appName, "-p", appDir, "--strategy=canary").Wait()
```

Comment:
[question, nonblocking] do we still want to test --no-wait somewhere

Reply:
I talked with @gururajsh and we decided that it would be ok for right now to skip the --no-wait test.